klippy/extras/display/st7920.py

```diff
@@ -108,20 +108,7 @@ def init(self):
                 0x06, # Set positive update direction
                 0x0c] # Enable display and hide cursor
         self.send(cmds)
-        # Setup animated glyphs
-        self.cache_glyph('fan2', 'fan1', 0)
-        self.cache_glyph('bed_heat2', 'bed_heat1', 1)
         self.flush()
-    def cache_glyph(self, glyph_name, base_glyph_name, glyph_id):
-        icon = icons.Icons16x16[glyph_name]
-        base_icon = icons.Icons16x16[base_glyph_name]
-        for i, (bits, base_bits) in enumerate(zip(icon, base_icon)):
-            pos = glyph_id*32 + i*2
-            b1, b2 = (bits >> 8) & 0xff, bits & 0xff
-            b1, b2 = b1 ^ (base_bits >> 8) & 0xff, b2 ^ base_bits & 0xff
-            self.glyph_framebuffer[pos:pos+2] = [b1, b2]
-            self.all_framebuffers[1][1][pos:pos+2] = [b1 ^ 1, b2 ^ 1]
-        self.cached_glyphs[glyph_name] = (base_glyph_name, (0, glyph_id*2))
```

Comment:
Instead of deleting this optimization, I'd make it conditional.  That is, change the first two lines to:
```
        icon = self.icons.get(glyph_name)
        base_icon = self.icons.get(base_glyph_name)
        if icon is None or base_icon is None:
            return
```
Then just add the following to the end of set_glyphs():
```
        self.cache_glyph('fan2', 'fan1', 0)
        self.cache_glyph('bed_heat2', 'bed_heat1', 1)
```
That way, the optimization is available for the common case, but isn't a requirement.